drivers/SmartThings/zigbee-switch/src/configurations.lua

```diff
@@ -63,7 +66,69 @@ local devices = {
       }
     },
     IAS_ZONE_CONFIG_METHOD = constants.IAS_ZONE_CONFIGURE_TYPE.AUTO_ENROLL_RESPONSE
-  }
+  },
+  FRIENT_SWITCHES = {
+    FINGERPRINTS = {
+      { mfr = "frient A/S", model = "SPLZB-131" },
+      { mfr = "frient A/S", model = "SPLZB-132" },
+      { mfr = "frient A/S", model = "SPLZB-134" },
+      { mfr = "frient A/S", model = "SPLZB-137" },
+      { mfr = "frient A/S", model = "SPLZB-141" },
+      { mfr = "frient A/S", model = "SPLZB-142" },
+      { mfr = "frient A/S", model = "SPLZB-144" },
+      { mfr = "frient A/S", model = "SPLZB-147" },
+      { mfr = "frient A/S", model = "SMRZB-143" },
+      { mfr = "frient A/S", model = "SMRZB-153" },
+      { mfr = "frient A/S", model = "SMRZB-332" },
+      { mfr = "frient A/S", model = "SMRZB-342" }
+    },
+    CONFIGURATION = {
+      {
+        cluster = ElectricalMeasurement.ID,
+        attribute = ElectricalMeasurement.attributes.RMSVoltage.ID,
+        minimum_interval = 5,
+        maximum_interval = 600,
+        data_type = ElectricalMeasurement.attributes.RMSVoltage.base_type,
+        reportable_change = 1
+      },{
+        cluster = ElectricalMeasurement.ID,
+        attribute = ElectricalMeasurement.attributes.RMSCurrent.ID,
+        minimum_interval = 5,
+        maximum_interval = 600,
+        data_type = ElectricalMeasurement.attributes.RMSCurrent.base_type,
+        reportable_change = 1
+      },{
+        cluster = ElectricalMeasurement.ID,
+        attribute = ElectricalMeasurement.attributes.ActivePower.ID,
+        minimum_interval = 5,
+        maximum_interval = 600,
```

Comment:
Our default max here is one hour, rather than 5 minutes. Is there a particular reason you've chosen a shorter max interval? Because of the reportable change, any updates to the value itself should still come in (provided they're not more frequent than the minimum interval), so we're wondering if there's a specific reason you want to see the current value reported every five minutes.

If not, we'd appreciate you sticking with our default configurations.